.travis.yml

```diff
@@ -1,13 +1,17 @@
+# Fix TravisCI, add github token
 language: php
 
+env:
+  - GITHUB_TOKEN=fbc9f8b635109c0c7ad64013ff8346908389f5f9
+
 matrix:
   include:
     - php: 5.4
     - php: 5.5
     - php: 5.6
     - php: 7
     - php: 7.1
-      env: WITH_CS=true
+      env: WITH_CS=true GITHUB_TOKEN=fbc9f8b635109c0c7ad64013ff8346908389f5f9
```

Comment:
Do we really want to expose a GitHub API token like that?

Reply:
This is a token without any selected permissions. But I think we should hide it.

Reply:
@tungvn 

If we encrypt it, it won’t be available for pull request builds of forks, so I think we can just leave it out. 

See https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml:

>Encrypted environment variables are not available to pull requests from forks due to the security risk of exposing such information to unknown code.
>
>If you define a variable with the same name in .travis.yml and in the Repository Settings, the one in `.travis.yml` takes precedence. If you define a variable in `.travis.yml` as both encrypted and unencrypted, the one defined later in the file takes precedence.